index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/a076d05399.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
+    <link rel="stylesheet" href="style/style.css">
+    <title>Portfolio</title>
+    <style>
+        section{
+            background: url("header-shapes mobile.png");
+            background-size: cover;
+        }
+    </style>
+</head>
+<body>
+    <header>
+        <nav>
+            <input type="checkbox" id="check">
+            <label for="check" class="checkbtn">
+                <i class="fas fa-bars"></i>
+            </label>
+            <label class="logo">My logo</label>
+            <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section>
+        <h1 class="title">I'm Salwa <br>Glad to see you!</h1>
+        <p class="presentation">I’m a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact
+            me.
+        </p>
+        <p class="connect">Let's connect</p>
+        <div class="social-icon">
+                <a ref="#"><img src="style/img/twitter.png" class="icon-image" alt="twitter icon"/></a>
+                <a ref="#"><img src="style/img/m.png" class="icon-image" alt="m icon"/></a>
+                <a ref="#"><img src="style/img/github.png" class="icon-image" alt="github icon"/></a>
+                <a href="#"><img src="style/img/linkedin.png" class="icon-image"></img></a>
+                <a href="#"><img src="style/img/v.png" class="icon-image" alt="v icon"/></a>
+                
+        </div>
```

Comment:
- [ ] It is required that the list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). Kindly implement this suggestion to this part of the portfolio.

Reply:
thank you @Qoosim  for the accurate description, I will fix the missing parts